package.json

```diff
@@ -2,11 +2,11 @@
   "name": "aframe-altspace-component",
   "version": "1.0.0",
   "description": "aframe-altspace-component makes A-frame apps compatible with AltspaceVR.",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
     "build": "browserify examples/main.js -o examples/build.js",
     "dev": "budo examples/main.js:build.js --dir examples --port 8000 --live --open",
-    "dist": "webpack index.js dist/aframe-altspace-component.js && webpack -p index.js dist/aframe-altspace-component.min.js",
+    "dist": "webpack src/index.js dist/aframe-altspace-component.js && webpack -p src/index.js dist/aframe-altspace-component.min.js",
```

Comment:
There's a `require('./sync-n-sound.js')` call in index.js. We should separate all the components out similarly.